bundle/Resources/translations/ezplatform_fields_groups.fr_FR.xlf

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" datatype="plaintext" original="file.ext" target-language="fr">
+    <body>
```

Comment:
sorry I wrote a tag in it I was speaking about the header it's present in all your files except this one